### ISSUE TITLE
Continue training after crash

### ISSUE THREAD

**Issue by IsiSku** [fchollet/keras#1688]
Hello,
I attempted training a covnet. Since it takes a long time, I figured something might happen with the computer and it might crash at some point, and it did. I did include the ModelCheckpoint callback so weights are saved in a .hdf5 file. Unfortunately, only a quarter of epochs passed and I would like to continue training the network.

I saw an answer on [stackoverflow](http://stackoverflow.com/questions/35074549/how-to-load-a-model-from-an-hdf5-file-in-keras) which says that I should define the layers as I would when beginning (passing the init='something') and after that loading the weights but it confuses me - wouldn't init and load_weights counteract each other?

I'm still new to this so would you mind giving me an example of loading weights from such file and initializing a network with it to continue training? I would not bother you but I didn't quite find the documentation helpful.

Thank you in advance!

**Comment by tboquet** [comment]
Hi, you could redefine your model and simply call:

```python
model = Sequential()

....

model.load_weights(filepath)
```

**Comment by isaacgerg** [comment]
I use a framework which saves the json and the weights every 5 minutes of training.  When something goes wrong and stops the training, I pass the json filename to my framework and it automatically builds the network, loads the weights, and starts where it left off.

**Comment by sunshineatnoon** [comment]
@isaacgerg I have a concern about retraining on checkpoints, say if I use `SGD + weight dacay`, if I start train on a checkpoint, will the `learning_rate` be the decayed `learning_rate` or the initial `learning_rate`?

**Comment by isaacgerg** [comment]
Do you mean checkpoints in the keras framework?  I wrote my own.

**Comment by sunshineatnoon** [comment]
So you manually changed the learning rate when continuing training ?

**Comment by isaacgerg** [comment]
In my framework yes.

**Comment by sunshineatnoon** [comment]
I see, Thanks!
